src/benchmarklib/benchmark_config.cpp

```diff
@@ -3,7 +3,7 @@
 namespace opossum {
 
 BenchmarkConfig::BenchmarkConfig(const BenchmarkMode benchmark_mode, const ChunkOffset chunk_size,
-                                 const EncodingConfig& encoding_config, const bool indexes, const size_t max_runs,
+                                 const EncodingConfig& encoding_config, const bool indexes, const int64_t max_runs,
```

Comment:
Do you think that we will ever exceed int32_t here?

Reply:
I wouldn’t be surprised if we did. Especially some long running self-pimping tests (or super slow lambda functions 😁) might run TPC-H Q6 for hours.
And it basically doesn’t cost anything.

Reply:
Alright.